ejenti/pulse_trigger.py

```diff
@@ -72,6 +74,13 @@ def main():
         logging.error('There is not command config. (Loaded from {})'.format(cmd_config_file))
         exit(1)
 
+    # loading server job config
+    server_job_config_arg = arguments['--server-job-config']
+    server_job_config_file = os.path.abspath(server_job_config_arg)
+    server_job_config = CommonUtil.load_json_file(server_job_config_file)
+    if not server_job_config:
+        logging.warn('There is not server job config, so, there is no server jobs. But trigger will still running. (Loaded from {})'.format(server_job_config_file))
```

Comment:
'There is no server job config included. So, there will be no server job, but trigger will still go on.'